From 6df15bdd922747671d0879a447c9b130a6d645fd Mon Sep 17 00:00:00 2001
From: Biwen Li <biwen.li@nxp.com>
Date: Tue, 19 Mar 2019 15:46:37 +0800
Subject: [PATCH 116/235] drivers: soc: fsl: fix problem of compiling qixis
 driver

select CONFIG_MFD_CORE for qixis driver:
	Qixis driver is depended by function devm_mfd_add_devices.
	Function devm_mfd_add_devices is defined in drivers/mfd/mfd-core.c.
	Compile system will compile drivers/mfd/mfd-core.c once enabled CONFIG_MFD_CORE.

Signed-off-by: Biwen Li <biwen.li@nxp.com>
[Xulin: Original patch taken from NXP LSDK-19.06.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/soc/fsl/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/soc/fsl/Kconfig b/drivers/soc/fsl/Kconfig
index aaf47201f9e3..03c9bd2a6ae3 100644
--- a/drivers/soc/fsl/Kconfig
+++ b/drivers/soc/fsl/Kconfig
@@ -22,6 +22,7 @@ config FSL_QIXIS
 	depends on OF
 	select REGMAP_I2C
 	select REGMAP_MMIO
+	select MFD_CORE
 	default n
 	help
 	  Say y here to enable QIXIS system controller api. The qixis driver
-- 
2.17.1

